zerver/lib/error_notify.py

```diff
@@ -194,11 +192,10 @@ def email_server_error(report: Dict[str, Any]) -> None:
     mail_admins(format_email_subject(email_subject), message, fail_silently=True)
 
 
-def do_report_error(type: str, report: Dict[str, Any]) -> HttpResponse:
+def do_report_error(type: str, report: Dict[str, Any]) -> None:
     if type == "browser":
         notify_browser_error(report)
     elif type == "server":
         notify_server_error(report)
     else:
         raise JsonableError(_("Invalid type parameter"))
-    return json_success()
```

Comment:
Yes, that's correct, but I think it still makes sense to convert it to an `AssertionError`, because `JsonableError` isn't appropriate for this context.